code/game/objects/structures/signs/signs_interactive.dm

```diff
@@ -82,3 +82,64 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/delamination_counter, 32)
 			. += span_info("Good work!")
 		if(11 to INFINITY)
 			. += span_info("Incredible!")
+
+GLOBAL_LIST_EMPTY(map_collision_counters)
+
+/obj/structure/sign/collision_counter
+	name = "incident counter"
+	sign_change_name = "Indicator board- Tram incidents"
+	desc = "A display that indicates how many tram related incidents have occured today."
+	icon_state = "tram_hits"
+	is_editable = TRUE
+	var/hit_count = 0
+	var/tram_id = TRAM_LIFT_ID
+
+/obj/structure/sign/collision_counter/Initialize(mapload)
+	. = ..()
+
+	return INITIALIZE_HINT_LATELOAD
+
+/obj/structure/sign/collision_counter/LateInitialize()
+	. = ..()
+	for(var/obj/structure/industrial_lift/tram/tram as anything in GLOB.lifts)
+		GLOB.map_collision_counters += src
+		RegisterSignal(tram, COMSIG_TRAM_COLLISION, .proc/new_hit)
+		update_appearance()
```

Comment:
A: don't add to the global list once per tram, do it once per initialize. Same applies for updating appearance
B: why not check tram_id?

Reply:
Fixed A, we don't need a global list anyways.
RegisterSignal(tram_id, COMSIG_TRAM_COLLISION, .proc/new_hit) doesn't register. tram_id is just "tram"